dotCMS/src/main/java/com/dotcms/analytics/AnalyticsAPI.java

```diff
@@ -0,0 +1,46 @@
+package com.dotcms.analytics;
+
+import com.dotcms.analytics.app.AnalyticsApp;
+import com.dotcms.analytics.model.AccessToken;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.Config;
+
+
+/**
+ * Analytics functionality interface.
+ *
+ * @author vico
+ */
+public interface AnalyticsAPI {
+
+    String ANALYTICS_IDP_URL_KEY = "analytics.idp.url";
+    String ANALYTICS_ACCESS_TOKEN_TTL_MINUTES_KEY = "analytics.access.token.ttl.minutes";
+    int ANALYTICS_ACCESS_TOKEN_TTL_MINUTES = Config.getIntProperty(ANALYTICS_ACCESS_TOKEN_TTL_MINUTES_KEY, 2) * 60;
```

Comment:
is this millis?
if it is, you can use DateUtil.MINUTE_MILLIS instead 60

Reply:
Got it.